signalflow/example/main.go

```diff
@@ -33,6 +38,12 @@ func main() {
 	fmt.Printf("Max Delay: %v\n", comp.MaxDelay())
 	fmt.Printf("Detected Lag: %v\n", comp.Lag())
 
+	go func() {
+		for msg := range comp.Expirations() {
+			fmt.Printf("Got expiration notice for TSID %s", msg.TSID)
```

Comment:
being nit picky here fmt.Printf will not to to your logger file instead will go to stdout which will make debugging harder

Reply:
This example is meant to be run on the terminal with `go run ...` it isn't meant to be a production app.